mmpose/datasets/datasets/mesh/mesh_h36m_dataset.py

```diff
@@ -27,7 +27,7 @@ class MeshH36MDataset(MeshBaseDataset):
             validation dataset. Default: False.
     """
 
-    def evaluate(self, outputs, res_folder, metric='joint_error'):
+    def evaluate(self, outputs, res_folder, metric='joint_error', **kwargs):
```

Comment:
where is this kwargs used

Reply:
If I remove **kwargs, it will cause an error in the eval hook. The eval hook use this function by:
`        eval_res = self.dataloader.dataset.evaluate(
            results, runner.work_dir, logger=runner.logger, **self.eval_kwargs)`
`logger` is passed into the evaluate function.

Reply:
this need a better solution. pls wait

Reply:
OK

Reply:
solution: put `logger=None` at the the argument list

Reply:
OK